### ISSUE TITLE
Exiting gracefully from Extract script?

### ISSUE THREAD

**Issue by cypherpunk1** [deepfakes/faceswap-playground#169]
It's possible to save model data upon manually exiting the Training script, but _**is there a feature available to exit the Extract procedure and still have it write to the alignments.json?**_ 
I've had a few instances where in the middle of a session I've wanted to adjust some settings (such as rotation degrees), or remove some unnecessary frames, but **Ctrl-C** causes it to quit abruptly without saving.

**Comment by bryanlyon** [comment]
Yes it is possible, but it is not implemented because we don't recommend partial extracts using different commands.  If you want to do this you can enable it by adding an exception capture in extract.py around line 37 to capture the ctrl-c and skip to the writing out.  I will not give the exact details of how to do this, since it's for experts only.
